### DISCUSSION TITLE
Import error message on page of broken DAG

### DISCUSSION THREAD

**Discussion by LvdKnaap** [apache/airflow#28104] [General]
### Description

For a DAG that has an import error, I would like to see an error message on the DAG-specific page

### Use case/motivation

Currently I see an Error message "DAG Import Errors (x)" on the home page with a dropdown menu specifying errors per broken DAG. 

Howerver, for broken dags listed here, there is no mention of an error on the DAG-specific page.

With multiple DAGs managed by multiple users, the Import Error message on the home page quickly becomes an oversight. 

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by tirkarthi** [comment]
Can you please add the airflow version? This PR seems to be relevant to this issue https://github.com/apache/airflow/pull/17818

**Comment by potiuk** [comment]
Very likely this is simply old Airflow Version. Converted to disciussion if more discussion needed.
